src/perf.ts

```diff
@@ -0,0 +1,74 @@
+'use strict';
```

Comment:
Please make the perf.ts as a standalone nodejs executable.
`node perf.ts`

Reply:
Ack.